diff --git a/dist/esm/server.js b/dist/esm/server.js
index 9b353530a5886b4b551d07bfb7e5bf920d7ec28e..b352d3e865c9fd5b0211b71dc27369172afa2703 100644
--- a/dist/esm/server.js
+++ b/dist/esm/server.js
@@ -217,7 +217,9 @@ async function handleManifestRequest(build, routes, url) {
     }
     return json(patches, {
       headers: {
-        "Cache-Control": "public, max-age=31536000, immutable"
+        "Cache-Control": "public, max-age=0, must-revalidate",
+        "CDN-Cache-Control": "public, max-age=31536000",
+        "Netlify-Vary": "query"
       }
     }); // Override the TypedResponse stuff from json()
   }
diff --git a/dist/server.js b/dist/server.js
index 9d0c9e443393499c0e9d5f764d8e4624ca02179e..ded7e467e20ef5dfb2cc8be32bfedf115322afea 100644
--- a/dist/server.js
+++ b/dist/server.js
@@ -221,7 +221,9 @@ async function handleManifestRequest(build, routes, url) {
     }
     return responses.json(patches, {
       headers: {
-        "Cache-Control": "public, max-age=31536000, immutable"
+        "Cache-Control": "public, max-age=0, must-revalidate",
+        "CDN-Cache-Control": "public, max-age=31536000",
+        "Netlify-Vary": "query"
       }
     }); // Override the TypedResponse stuff from json()
   }
